HodowlaYorkland/Content/myStyle.css

```diff
@@ -5,3 +5,46 @@
     background-color: #2f3336;
     color: #efefef;
 }
+
+h1.logo{
```

Comment:
Each element should have one space between identifier and `{`, for example `h1.logo {`